### DISCUSSION TITLE
why the entry scss file did not generate to distinate css file

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [webpack/webpack#15254] [Q&A]
I am using webpack 5.x to generate a entry scss file but failed, this is the detail information:

https://stackoverflow.com/questions/70851517/why-the-webpack-5-x-did-not-generate-entry-scss-file-to-css-file

what should I do to make it generate the css file from entry?

**Comment by alexander-akait** [comment]
Please remove `style-loader`, if you need extract CSS
